### PR TITLE
Lars review

### PR DESCRIPTION
**Pull request by martinduke**

### CODE REVIEW

File: draft-ietf-shmoo-cancel-meeting.md [hunk 1]
```diff
@@ -34,13 +34,13 @@ infeasible. This document provides criteria for making this judgment.
 
 # Introduction
 
-One highlight of the IETF calendar is in-person general meetings, which happen
-three times a year at various locations around the world.
+Among the highlights of the IETF calendar is in-person general meetings, which
```
Comment (larseggert) [inline draft-ietf-shmoo-cancel-meeting.md]:
```suggestion
Among the highlights of the IETF calendar are in-person general meetings, which
```

File: draft-ietf-shmoo-cancel-meeting.md [hunk 2]
```diff
@@ -149,8 +150,9 @@ later update of this document.
 ## IESG and IRTF Chair
 
 If the LLC assesses there are no fundamental logistical or financial obstacles
-to holding the meeting, the IESG and IRTF chair assess if projected attendance
-is high enough to capture the benefit of an in-person meeting.
+to holding a meeting in an emergency situation, the IESG and IRTF chair assess
```
Comment (larseggert) [inline draft-ietf-shmoo-cancel-meeting.md]:
```suggestion
to holding a meeting in an emergency situation, the IESG and IRTF Chair assess
```

File: draft-ietf-shmoo-cancel-meeting.md [hunk 3]
```diff
@@ -165,11 +167,11 @@ remote?
 
 # Remedies
 
-If the event cannot be held at the scheduled time and place, the IETF has
-several options. The remedies below should be consdered in light of these
-principles, presented in no particular order:
+If an event cannot be held at the scheduled time and place, the LLC and IESG
```
Comment (larseggert) [inline draft-ietf-shmoo-cancel-meeting.md]:
```suggestion
If a meeting cannot be held at the scheduled time and place, the LLC and IESG
```

Comment (larseggert) [reply draft-ietf-shmoo-cancel-meeting.md]: Could you also check other uses of "event" vs. "meeting" throughout the document? A "meeting" is what we hold, an "event" is the thing that might cause us to cancel a meeting (at least I think that is how most of the text is written.)

File: draft-ietf-shmoo-cancel-meeting.md [hunk 4]
```diff
@@ -149,8 +150,9 @@ later update of this document.
 ## IESG and IRTF Chair
 
 If the LLC assesses there are no fundamental logistical or financial obstacles
-to holding the meeting, the IESG and IRTF chair assess if projected attendance
-is high enough to capture the benefit of an in-person meeting.
+to holding a meeting in an emergency situation, the IESG and IRTF chair assess
+if projected attendance is high enough to achieve the benefit of an in-person
+meeting.
 
 The IESG is discouraged from relying on a simple head count of expected event
```
Comment (larseggert) [inline draft-ietf-shmoo-cancel-meeting.md]:
```suggestion
The IESG is discouraged from relying on a simple head count of expected meeting
```

File: draft-ietf-shmoo-cancel-meeting.md [hunk 5]
```diff
@@ -149,8 +150,9 @@ later update of this document.
...
 attendance. Even dramatically smaller events with large remote participation may
```
Comment (larseggert) [inline draft-ietf-shmoo-cancel-meeting.md]:
```suggestion
attendance. Even dramatically smaller meetings with large remote participation may
```

**Review by larseggert — CHANGES_REQUESTED** [review]

File: draft-ietf-shmoo-cancel-meeting.md (hunk 3 above)
Comment (martinduke) [reply draft-ietf-shmoo-cancel-meeting.md]: OK, it's done. If you're now happy with it, I'll merge this and publish a new version.

**Review by larseggert — APPROVED** [review]

**Pull request merged**